packages/cli/__tests__/zostso/__system__/stop/__snapshots__/cli.zos-tso.stop.system.test.ts.snap

```diff
@@ -1,13 +1,17 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`zos-tso stop should fail with invalid option 1`] = `
-"Command Error:
+"Unable to perform this operation due to the following problem.
 Unknown arguments: foo-bar, fooBar
 Command failed due to improper syntax
 Command entered: \\"zos-tso start as --foo-bar\\"
 Available commands are \\"address-space\\".
 Use \\"zowe zos-tso start --help\\" to view groups, commands, and options.
-Error Details:
+
+Response From Service
+Error: Unknown arguments: foo-bar, fooBar
+
+Diagnostic Information
```

Comment:
It seems odd to me to show the "Unknown arguments" error in the "Response From Service" category. There is no response from the service if client-side validation fails before a request is issued. And the error message is duplicated in the "Diagnostic Information" below. Is this an intended behavior of the new error formatting in v3?

Reply:
The error format handling typically takes properties from a REST response (or an imperative error) and places them under the most appropriate titles. When we never get to the remote service, values are filled in with the available information.

In the case of the zos-tso command run in this test, here is the output from running the same command interactively:
```
-> zowe zos-tso start as --foo-bar
Unable to perform this operation due to the following problem.
Unknown arguments: foo-bar, fooBar
Command failed due to improper syntax
Command entered: "zos-tso start as --foo-bar"
Available commands are "address-space".
Use "zowe zos-tso start --help" to view groups, commands, and options.

Response From Service
Error: Unknown arguments: foo-bar, fooBar

Diagnostic Information
Unknown arguments: foo-bar, fooBar
```
I feel that the objectives of this PR to remove V1 profile definitions (and fix the inability of system test scripts to properly detect newly formatted errors) are being met. I think that additional changes should be handled outside this PR.

It might be valid to open an issue about how the zos-tso command group prepares its error message content (or how the error formatting logic choses to display that content).

Reply:
This is the output of the command in the old error format:
```
$ zowe zos-tso start as --foo-bar
Command Error:
Unknown arguments: foo-bar, fooBar
Command failed due to improper syntax
Command entered: "zos-tso start as --foo-bar"
Available commands are "address-space".
Use "zowe zos-tso start --help" to view groups, commands, and options.
Error Details:
Unknown arguments: foo-bar, fooBar
```